src/data/nat/order/basic.lean

```diff
@@ -298,6 +277,14 @@ end
 | 0 := iff_of_false (lt_irrefl _) zero_le_one.not_lt
 | (n + 1) := lt_mul_iff_one_lt_left n.succ_pos
 
+lemma add_sub_one_le_mul {a b : ℕ} (ha : a ≠ 0) (hb : b ≠ 0) : a + b - 1 ≤ a * b :=
```

Comment:
Maybe unnecessary, but you could also add the following:
```
lemma add_sub_one_le_mul' {a b : ℕ} (ha : a ≤ 1) (hb : b ≤ 1) : a + b - 1 ≤ a * b := sorry
```

Reply:
That looks a bit useless. I'm adding `add_sub_one_le_mul` because I use it twice, but I wouldn't even be writing it down if we had a working `omega`.